packages/formation-react/src/components/AlertBox/AlertBox.jsx

```diff
@@ -32,12 +32,16 @@ class AlertBox extends Component {
   }
 
   scrollToAlert = () => {
+    if (!this._ref || !this._ref.scrollIntoView) {
```

Comment:
Alternatively, you can use optional chaining.
```suggestion
    if (!this._ref?.scrollIntoView) {
```